sprout/Dockerfile

```diff
@@ -4,9 +4,9 @@ MAINTAINER maintainers@projectclearwater.org
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes sprout
 RUN sed -e 's/\(echo 0 > \/proc\/sys\/kernel\/yama\/ptrace_scope\)/# \0/g' -i /etc/init.d/sprout
 
-COPY chronos.supervisord.conf /etc/supervisor/conf.d/chronos.conf
-COPY memcached.supervisord.conf /etc/supervisor/conf.d/memcached.conf
 COPY sprout.supervisord.conf /etc/supervisor/conf.d/sprout.conf
 COPY clearwater-group.supervisord.conf /etc/supervisor/conf.d/clearwater-group.conf
+COPY plugins/* /usr/share/clearwater/clearwater-cluster-manager/plugins/
```

Comment:
@rkd-msw No -- this is my new memcached-user-plugin.   It's only needed in Docker and so I've defined it here in the clearwater-docker repo?